.github/workflows/ci_pullrequest.yml

```diff
@@ -341,19 +324,20 @@ jobs:
       - name: "Install MySql Dependencies"
         run: "poetry install --no-interaction --no-root --extras mysql"
       - name: "Run migration test"
-        run: "poetry run invoke migration-test --db-engine mysql --dataset ${{ matrix.dataset-file }}"
+        run: "poetry run invoke migration-test --db-engine mysql --dataset $MIGRATION_TEST_DATASET"
     needs:
-      - "pylint"
-      - "tests-mysql"
-
+      - "black"
+      - "check-migrations"
+      - "check-schema"
+      - "flake8"
+      - "hadolint"
+      - "markdownlint"
+      - "yamllint"
   migration-tests-postgres:
+    name: "Run migrations against test dataset (PostgreSQL and Python 3.10)"
```

Comment:
Quick question, Should we run this using Python 3.11? Since that is what we are doing with the unit tests.

Reply:
Probably. Not sure if this was missed or intentional when we upped the max version.